morse_module/morse.c

```diff
@@ -146,6 +170,16 @@ static inline void setup_mcu() {
 	gpio_pull_down(FREQ_INC);
 	gpio_pull_down(TX);
 
+	// Setting up the addr bits
+	int pins[4] = {ADDR0, ADDR1, ADDR2, ADDR3};
+	for(int i = 0; i < 4; i++) {
```

Comment:
use `size_t` for `i`, and use a const instead of the `4`, either `sizeof(pins)/sizeof(pins[0])` or a const that you define